ppr-ui/src/components/mhrRegistration/HomeOwners/AddHomeOwnerOrganization.vue

```diff
@@ -0,0 +1,227 @@
+<template>
+  <div>
```

Comment:
it looks like you can get away with the `<v-card>` as your root element here.

Reply:
Updated